packages/webapp-libs/webapp-api-client/scripts/download-graphql-schema.js

```diff
@@ -0,0 +1,33 @@
+const { spawn } = require('child_process');
+const fs = require('fs-extra');
+const dotenv = require('dotenv');
+
+function runCommand(command, args) {
+  return new Promise((resolve, reject) => {
+    const cmd = spawn(command, args, { stdio: 'inherit' });
+
+    cmd.on('close', (code) => {
+      if (code !== 0) {
+        reject(new Error(`"${command} ${args.join(' ')}" failed with code ${code}`));
+      } else {
+        resolve();
+      }
+    });
+  });
+}
+
+(async () => {
+  try {
+    dotenv.config({ path: '../../webapp/.env' });
+
+    const apiUrl = 'http://localhost:5001/api/graphql/';
+
+    fs.remove('./src/graphql/__generated/types.ts');
```

Comment:
I think this should be `fs.removeSync`

Reply:
Or just a missing `await`, since those return promises